package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "eigengraus-essential-establishment-generator",
+  "version": "0.1.0",
```

After digging around in the code a bit, I can see that there exists a changelog (under `Meta/ChangeLogFull.twee`) with the newest version being `2.2.1`
```suggestion
  "version": "2.2.1",
```